ui/src/copainput.tsx

```diff
@@ -63,6 +75,19 @@ export function CopaInput(props: any) {
       setSelectedImageHelperText("Image input can not have whitespace.")
     }
 
+    let seperateSplit = props.selectedImage.split(':');
+    let numColons = seperateSplit.length - 1;
+
+    if (numColons > 1) {
+      foundError = true;
+      setSelectedImageHelperText("Image input can only have one colon.");
+    } else {
+      if (seperateSplit[0].length === 0) {
+        foundError = true;
+        setSelectedImageHelperText("Image input can not be a tag only.")
+      }
```

Comment:
In the chance that the input is completely empty, I think it would still show "input cannot be tag only" which could be a bit misleading

Reply:
Yes, this is what happens. I'll make sure to add a check for an empty string before we check for anything related to tags.